bump/action.yml

```
@@ -45,6 +45,12 @@ inputs:
     description: 'Optional SemVer prefix (eg. "", "v", "componentX-"). If set, only tags with this exact prefix shall be considered. "*" is a special value, meaning the closest SemVer is used, regardless of prefix".'
```

```suggestion
    description: 'Optional version prefix (eg. "", "v", "componentX-"). If set, only tags with this exact prefix shall be considered. "*" is a special value, meaning the closest version is used, regardless of prefix".'
```
As this prefix seems to be used now in both schemes (sdkver and semver) effectively.